rasa/nlu/classifiers/diet_classifier.py

```diff
@@ -880,6 +881,7 @@ def _predict_label(
         # if X contains all zeros do not predict some label
         if label_ids.size > 0:
             label = {
+                "id": hash(self.index_label_id_mapping[label_ids[0]]),
```

Comment:
for my understanding: That's the hash of the intent name then, correct?

Reply:
nope hash of the text...but this will change again once #6392 gets merged...

Reply:
That's within the `DietClassifier` so I think it's the intent, no? Anyway, not important if get's changed by Daksh's PR